scripts/generateAgencyData.js

```diff
@@ -1,28 +1,24 @@
 /* eslint-disable no-console, arrow-body-style */
-const childProcess = require('child_process')
-const fs = require('fs')
-const path = require('path')
+import childProcess from 'child_process'
```

Comment:
I don't think we should use `import` here because it isn't supported in node version 6 (which is what our project is currently using). I'd recommend using require and submitting a new PR that updates the node version to 8